aries-site/src/pages/whats-new.mdx

```diff
@@ -52,14 +52,87 @@ regarding either via [Slack](https://hpe.slack.com/archives/C04LMJWQT).
+   - Completed documentation around the definition, scope and anatomy of Page Layouts.
+   - Created guidelines for the Page Container, including defining 3 types of content widths: Wide, Narrow and Full.
+   - Defined new breakpoints and page margins for each breakpoint.
+   - Work in progress for Page Layout templates, starting with 2 column, 3 column and multi-column Dashboards.
+
+### Stay current with the latest changes
+- **Voice and Tone** - Updates in progress for Voice & Tone page.
+- **Accessibility** - Updates in progress for Accessibility page to provide more resources and guidance.
+- **Persistent Sidebar to be deprecated by end of year** - In preparation for the deprecation of the Persistent Sidebar in Design System, we have removed any examples and documentation surrounding its use.
+
+### Latest Releases
+- **Grommet v2.20.0** - Release includes the NEW Tag component along with enhancements to Calendar, Carousel, DateInput, Form, Video; fixes applied to Anchor, Button, Calendar, Clock, Select, Text, and TextInput. Full details about this release are available in the [Release Notes](https://github.com/grommet/grommet/releases/tag/v2.20.0).
+
+## December 24 - January 3, 2022
+
+Happy Holidays 🎉
+
+## December 20 - 23, 2021
+
+### Roadmap Updates
+- **Page Layouts** - Commenced explorations on [Dashboards within Page Layouts](https://www.figma.com/file/4rdBkUlHd5MCVq3hvUOXHd/Page-Layouts?node-id=2159%3A17698).
+- **NameValueList** - Created variant of [editable horizontal form fields with NameValue lists](https://design-system.hpe.design/components/namevaluelist?q=name#name-value-pair-as-a-read-only-display-for-text-inputs).
+- **Navigation** - Commenced research for [Ascending Navigation](https://www.figma.com/file/b6hiokanRCwNnc93PpHdgQ/HPE-Design-System-Navigation?node-id=2382%3A13827).
+
+### Stay current with the latest changes
+- **Toast Notifications** - Updated guidance on [Toast Notifications](https://design-system.hpe.design/templates/toast-notifications)
+   - This includes [when a user should use a Toast notification](https://design-system.hpe.design/templates/toast-notifications#when-should-you-use-a-toast-notification), as well as when not to use a Toast Notification. There was more information given about the [accessibility](https://design-system.hpe.design/templates/toast-notifications#accessibility) behind Toast Notification.
+
+## December 6 - December 17, 2021
+
+### Roadmap Updates
+- **Button** 
+   - Completed guidance on placement of the [Cancel Button in a Form and Modal](https://design-system.hpe.design/components/button?q=but#styling).
+   - Updated Figma with the new hover states for Primary and Color Buttons and published within the [HPE theme](https://github.com/grommet/grommet-theme-hpe/pull/230).
+   - Updated Button Figma file on Cancel Buttons to use [default styling in Forms and Modals](https://design-system.hpe.design/components/button#cancel-buttons).
```

Comment:
```suggestion
   - Updated Button Figma file on [Cancel Buttons](/components/button#cancel-buttons) to use `default` styling in Forms and Modals instead of `secondary`.
```